api/src/main/java/org/openmrs/module/ohricore/engine/OHRIComputedConceptsManager.java

```diff
@@ -0,0 +1,16 @@
+package org.openmrs.module.ohricore.engine;
+
+import org.openmrs.Encounter;
+import org.openmrs.Patient;
+
+/**
+ * @author MayanjaXL, Amos, Stephen, smallGod date: 28/06/2021
+ */
+public class OHRIComputedConceptsManager {
```

Comment:
What is this class for? Remove it since the implementations are empty

Reply:
Okay, deleted this.